Composer/packages/tools/language-servers/language-generation/src/builtinFunctionsMap.ts

```diff
@@ -699,6 +684,7 @@ export const buildInfunctionsMap: Map<string, FunctionEntity> = new Map<string,
       'Return the first non-null value from one or more parameters. Empty strings, empty arrays, and empty objects are not null.'
     ),
   ],
+  /* C# only
```

Comment:
We can comment out those functions out for the reason that, those are not likely to be used, but not for c# only.  
because 1. C# works in Composer, because the runtime composer leveraged today is C#.  2. We should figure out in Expression side, why we are having C# only functions.